### PR TITLE
minor - code readability HTTP CODES as defines && clear warnings

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
Just a minor code cleanup for better readability of code that I didn't want to include in UTF8 URL parser PR due to IMHO being unrelated

##### Component Name
libnetdata/server

##### Additional Info
No behavior change. Just code readability - something that annoyed me a bit when I was going trough code. *(not all people keep all the codes in memory (besides 200/400/404))*
Additionally one compiler warning fixed if compiling with `-Wpointer-sign`

### CODE REVIEW

**Comment by thiagoftsm** [comment]
Hi @underhood ,

I liked what the PR you brought, without any doubt, the code will be more readable with these changes, but before we approve, I will ask you to change one thing that we missed in the previous PR, because I had a problem in my compiler and I could not detect it. As you can see below

/home/thiago/Netdata/netdata/libnetdata/url/url.c: In function ‘url_decode_r’:
/home/thiago/Netdata/netdata/libnetdata/url/url.c:211:29: warning: pointer targets in passing argument 1 of ‘utf8_check’ differ in signedness [-Wpointer-sign]
  211 |     if(unlikely( utf8_check(to) )) //NULL means sucess here
      |                             ^~
      |                             |
      |                             char *
/home/thiago/Netdata/netdata/./config.h:356:41: note: in definition of macro ‘unlikely’
  356 | #define unlikely(x) __builtin_expect(!!(x), 0)
      |                                         ^
/home/thiago/Netdata/netdata/libnetdata/url/url.c:126:42: note: expected ‘unsigned char *’ but argument is of type ‘char *’
  126 | unsigned char *utf8_check(unsigned char *s)
      |                           ~~~~~~~~~~~~~~~^

It is necessary to cast the variable when we call utf8_check, but for these new code that you are bringing  there is nothing to change.

Thanks a lot!

File: libnetdata/url/url.c [hunk 1]
```diff
@@ -208,7 +208,7 @@ char *url_decode_r(char *to, char *url, size_t size) {
 
     *d = '\0';
 
-    if(unlikely( utf8_check(to) )) //NULL means sucess here
+    if(unlikely( utf8_check((unsigned char *)to) )) //NULL means sucess here
```
Comment (ilyam8) [inline libnetdata/url/url.c]: > minor - code readability HTTP CODES as defines 
> No behavior change. Just code readability

Why do not add it as a separate PR? Or change the title/OP.

Comment (underhood) [reply libnetdata/url/url.c]: did it on request of @thiagoftsm

Comment (ilyam8) [reply libnetdata/url/url.c]: I know, but now the PR changes dont reflect title/OP.

Comment (underhood) [reply libnetdata/url/url.c]: OK, so do you want me to kick that commit out and make it separate PR *(cleaner, or let it be part of Url parser refactoring #6247)* or just change a title here

Comment (ilyam8) [reply libnetdata/url/url.c]: I think better to split not related things. It feels natural for me. However title/op should always reflect changes.

This was a comment, so just keep this in mind :+1:

**Comment by underhood** [comment]
> 
> 
> Damn, we just got a conflict due to a different PR getting merged. Can you please rebase?

Yep, the URL rework PR also fixed the comp warning in url.c. Will do when i get home

**Comment by underhood** [comment]
I kicked out that url.c change
